Source/Scene/Cesium3DTile.js

```diff
@@ -500,54 +513,70 @@ define([
         return this._boundingVolume.distanceToCamera(frameState);
     };
 
-    function createBoundingVolume(boundingVolumeHeader) {
-        var volume;
-        if (boundingVolumeHeader.box) {
+    var scratchMatrix = new Matrix3();
+    var scratchScale = new Cartesian3();
+    var scratchHalfAxes = new Matrix3();
+    var scratchCenter = new Cartesian3();
+    var scratchRectangle = new Rectangle();
+
+    function createBoundingVolume(boundingVolumeHeader, transform, result) {
```

Comment:
Up to you, but you might want to expose this (still `@private`) for fine-grained unit testing.